libs/trustchain/src/test-scenarios/removedMemberEjectedOnDeletedTrustchain.ts

```diff
@@ -0,0 +1,35 @@
+import { getEnv } from "@ledgerhq/live-env";
+import Transport from "@ledgerhq/hw-transport";
+import { getSdk } from "..";
+
+export async function scenario(transport: Transport) {
+  const applicationId = 16;
+
+  // first member initializes itself
+  const name1 = "Member 1";
+  const sdk1 = getSdk(!!getEnv("MOCK"), { applicationId, name: name1 });
+  const member1creds = await sdk1.initMemberCredentials();
+  // second member initializes itself
+  const name2 = "Member 2";
+  const sdk2 = getSdk(!!getEnv("MOCK"), { applicationId, name: name2 });
+  const member2creds = await sdk2.initMemberCredentials();
+  const member2 = { name: name2, id: member2creds.pubkey, permissions: 0xffffffff };
```

Comment:
As we use this on every test, we could make an helper to create that no?

Reply:
yes probably.
like https://github.com/LedgerHQ/ledger-live/pull/7262 has done it for the sdkForName one 👍